scipy-notebook/Dockerfile

```diff
@@ -60,6 +60,17 @@ RUN git clone https://github.com/PAIR-code/facets.git && \
 # Import matplotlib the first time to build the font cache.
 ENV XDG_CACHE_HOME="/home/${NB_USER}/.cache/"
 
+ARG CUSTOM_FONT=""
```

Comment:
Better variable name would be CUSTOM_MATPLOTLIB_FONT, because I think this only changes the font for matplotblib, doesn't it?